docs/includes/file-tutorial-excel-custom-functions.md

```diff
@@ -43,30 +43,44 @@ You’ll begin this tutorial by using the Yo Office generator to create the file
     * Choose a script type: `JavaScript`
     * What do you want to name your add-in? `stock-ticker`
 
-    ![Yo Office bash prompts for custom functions](../images/yo-office-cfs-stock-ticker-3.png)
+    ![Yo Office bash prompts for custom functions](../images/12-10-fork-cf-pic.jpg)
 
     After you complete the wizard, the generator will create the project files and install supporting Node components. The project files come from the [Excel-Custom-Functions](https://github.com/OfficeDev/Excel-Custom-Functions) GitHub repository.
 
 2. Navigate to the project folder.
 
-    ```bash
+    ```
     cd stock-ticker
     ```
 
-3. Start the local web server.
+3. Trust the self-signed certificate that is needed to run this project. For detailed instructions for either Windows or Mac, see [Adding Self Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md).  
+
+4. Build the project.
+
+    ```
+    npm run build-dev
+    ```
+    
+    After running this, you will see a readout in your command prompt about the build process. Note that you will be using this command later to re-build whenever you make changes to your functions files.
+
+5. Start the local web server, which runs in Node.js. 
 
     * If you'll be using Excel for Windows to test your custom functions, run the following command to start the local web server, launch Excel, and sideload the add-in:
 
-        ```bash
-        npm run start-desktop
+When running this command, you will see instructions to run `npm run start` however, run the below command instead: 
+         ```
+         npm run start-desktop
         ```
+        After running this command, your command prompt will show details about what has been done, another npm window will open showing the details of the build, and Excel will start with your add-in loaded. If you add-in does not load, check that you have completed step 3 properly.  
 
     * If you'll be using Excel Online to test your custom functions, run the following command to start the local web server: 
 
-        ```bash
+        ```
         npm run start-web
         ```
 
+         After running this command, another window will open showing you the details of the build. To use your functions, open a new workbook in Office Online. 
+
 ## Try out a prebuilt custom function
 
 The custom functions project that you created by using the Yo Office generator contains some prebuilt custom functions, defined within the **src/functions/functions.js** file. The **manifest.xml** file in the root directory of the project specifies that all custom functions belong to the `CONTOSO` namespace.
```

Comment:
The TypeScript version of the template uses *.ts extensions, so the path should be src\functions\functions.**ts**. For the JavaScript version of the template it is functions.js.

Reply:
It was pointed out this is specifically for the JavaScript version so this is OK.